### PR TITLE
Nina/add evaluation

### PR DESCRIPTION
**Pull request by ja-nina**
Added evaluation module for simpler evaluation running:
- wandb init put to training (else would be initialised for eval)
- evaluation module added
- evaluation given in set structures, which can be reused for analysis

### CODE REVIEW

File: MammoNet/models/BaseModel.py [hunk 1]
```diff
@@ -52,6 +54,9 @@ def validate(self, val_loader):
         return running_loss / len(val_loader), accuracy
 
     def train(self, train_loader, val_loader, test_loader):
+        # init wandb
+        setup_wandb("mammonet_project", {"model": self.model_name, "epochs": self.epochs})
```
Comment (eniseirem) [inline MammoNet/models/BaseModel.py:58]: ok it's logical to add here. nicely done!

File: MammoNet/evaluate/evaluator.py
Comment (eniseirem) [inline MammoNet/evaluate/evaluator.py:1]: maybe auc could be added later for report purposes.

**Review by eniseirem — APPROVED** [review]
project got so much better with the updates thanks!

File: MammoNet/evaluate/evaluator.py
Comment (ja-nina) [reply MammoNet/evaluate/evaluator.py:1]: Good idea!

**Pull request merged**